.github/workflows/ci-build.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
         with:
-          python-version: '3.12'
+          python-version: '3.13'
```

Comment:
@joelvdavies @joshdimanteto I don't think this step is needed anymore given the python services are running in Docker containers now.